channel_setup/import_channels.sh

```diff
@@ -1,5 +1,25 @@
 #!/bin/bash
 
+if [ "$1" = "-h" ] || [ "$1" = "--help" ]; then
+  echo "NAME"
+  echo "  import_channels_network - Import numeracy channels"
+  echo
+  echo "SYNOPSIS"
+  echo "  ./import_channels [CHANNEL_ID]"
+  echo
+  echo "DESCRIPTION"
+  echo "  With no CHANNEL_ID supplied, the script imports all numeracy channels from the "
```

Comment:
This function takes in a path to a directory containing channels, not a channel_id